### ISSUE TITLE
Vend Corretto through SDKMan

### ISSUE THREAD

**Issue by EricEdens** [corretto/corretto-8#11]
(Split from https://github.com/corretto/corretto-8/issues/2):

[SDKMan!](https://sdkman.io/) could be used to install Amazon Corretto in parallel to other JDKs on Windows, macOS, and Linux, but it would require Corretto to be distributed as a tar.gz or ZIP archive:
https://github.com/sdkman/sdkman-db-migrations/blob/f92fe573d835706e837485dcf937a923b66fa488/src/main/scala/io/sdkman/changelogs/JavaMigrations.scala#L234-L241

**Comment by marc0der** [comment]
Please let me know if I can help in anyway to get your binaries deployed on our platform!

**Comment by mdeinum** [comment]
After the announcement on Devoxx I actually went ahead and created a patch for SDKman to include these versions. But as the current distribution formats aren't supported this is currently not possible.

See https://github.com/sdkman/sdkman-db-migrations/pull/44

**Comment by marc0der** [comment]
@mdeinum once the zip/tarball binaries become available you can simply update your PRs and we can merge.

**Comment by vijayyadav06** [comment]
any updates on the GA release date?

**Comment by alvdavi** [comment]
@marc0der @mdeinum zip/tarball versions of Corretto8 preview2 are now available

**Comment by marc0der** [comment]
@arun-gupta Thanks for making this happen!

**Comment by arun-gupta** [comment]
Reopening to discuss the ongoing support

**Comment by marc0der** [comment]
In the meanwhile, community members are raising [PRs like this one](https://github.com/sdkman/sdkman-db-migrations/pull/63) to keep Corretto up to date on SDKMAN.

**Comment by arun-gupta** [comment]
@marc0der FYI https://github.com/sdkman/sdkman-website/issues/29

**Comment by madeupname** [comment]
Hi, I just tried installing Corretto 8 and 16 and discovered that nothing in /bin has execute permissions. I first reported this here:

https://github.com/sdkman/sdkman-cli/issues/958

PATH is updated, but I guess there is no smoke/acceptance test that verifies "which java" points to the correct executable. Was scratching my head trying to figure out why the shell wass ignoring the shell PATH and using the one in Windows. Other sdkman distributions don't have this issue. If it's something I'm doing wrong, glad to be corrected.
